admin_manual/installation/nginx_examples.rst

```diff
@@ -56,16 +56,31 @@ your NGINX installation.
   server {
       listen 80;
       server_name cloud.example.com;
+
+      # needs to be served via HTTP
+      location /.well-known/acme-challenge { }
```

Comment:
Not quite sure if this will work, we might also just want to drop this completely as this is nothing ownCloud specific and people should ask at the Lets Encrypt community how to configure this.